### ISSUE TITLE
Simple and Multiple Linear Regression in Python

### ISSUE THREAD

**Issue by abhi386** [abhi386/ML-and-DS-articles#2]
- TL;DR - Simple and Multiple Linear Regression in Python

https://towardsdatascience.com/simple-and-multiple-linear-regression-in-python-c928425168f9
https://towardsdatascience.com/@adi.bronshtein
Key Takeaways - Interpretation of the Output table of the linear regression in statmodel

- Useful Code Snippets - 

import statsmodels.api as sm # import statsmodels 

X = df["RM"] ## X usually means our input variables (or independent variables)
y = target["MEDV"] ## Y usually means our output/dependent variable
X = sm.add_constant(X) ## let's add an intercept (beta_0) to our model

# Note the difference in argument order
model = sm.OLS(y, X).fit() ## sm.OLS(output, input)
predictions = model.predict(X)

# Print out the statistics
model.summary()

Useful Tools - none
Comments/Questions - in progress
What do you like about this article? What you wish to learn more from?
in progress
What issues you have while trying to follow the codes or setup in the article?
in progress
What parts of the articles that you are puzzling about?
in porgress
